src/lib/wrapAISDK.ts

```diff
@@ -0,0 +1,229 @@
+import { NodeTracerProvider, BatchSpanProcessor } from '@opentelemetry/sdk-trace-node';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+import { defaultResource, resourceFromAttributes } from '@opentelemetry/resources';
+import { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION } from '@opentelemetry/semantic-conventions';
+import { Tracer } from '@opentelemetry/api';
+
+/**
+ * Configuration for the Scorecard AI SDK wrapper
+ */
+interface ScorecardConfig {
+  /**
+   * Scorecard project ID
+   * Defaults to SCORECARD_PROJECT_ID environment variable
+   */
+  projectId?: string;
+
+  /**
+   * Metrics to track
+   * Defaults to empty array
+   */
+  metrics?: string[];
+
+  /**
+   * Scorecard API key for authentication
+   * Defaults to SCORECARD_API_KEY environment variable
+   */
+  apiKey?: string;
+
+  /**
+   * Scorecard tracing endpoint
+   * Defaults to https://tracing.scorecard.io/otel/v1/traces
+   */
+  endpoint?: string;
+
+  /**
+   * Scorecard API base URL
+   * Defaults to https://api2.scorecard.io
+   */
+  apiBaseUrl?: string;
+
+  /**
+   * Service name for telemetry
+   * Defaults to "ai-sdk-app"
+   */
+  serviceName?: string;
+
```

Comment:
We can get rid of `endpoint` and `apiBaseUrl` and put them into the scorecard SDK options variable:

```suggestion
  // import ClientOptions from scorecard-ai

   scorecardSDKOptions?: Partial<Omit<ClientOptions, "apiKey">>
```

Reply:
I've ended up just removing the options all together since we are the only ones that would ever need it. When we do, we can just use env vars.